PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -10,33 +10,43 @@
 
 public class StartActivity extends Activity {
 
-	@Override
-	protected void onCreate(Bundle savedInstanceState) {
-		super.onCreate(savedInstanceState);
-		setContentView(R.layout.activity_main);
-		SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
-		boolean previouslyStarted = prefs.getBoolean(getString(R.string.pref_previously_started), false);
-		if (!previouslyStarted) {
-			startActivity(new Intent(StartActivity.this, AvatarRoomActivity.class));
-		}
-		ImageButton new_user = (ImageButton) findViewById(R.id.newUserButtonFirstPage);
-		new_user.setOnClickListener(new View.OnClickListener() {
-			@Override
-			public void onClick(View v) {
-				Intent myIntent = new Intent(StartActivity.this,
-						AvatarRoomActivity.class);
-				startActivityForResult(myIntent, 0);
-			}
-		});
-
-		ImageButton start = (ImageButton) findViewById(R.id.startButtonMain);
-		start.setOnClickListener(new View.OnClickListener() {
-			@Override
-			public void onClick(View v) {
-				Intent myIntent = new Intent(StartActivity.this, MapActivity.class);
-				startActivityForResult(myIntent, 0);
-			}
-		});
-
-	}
+    SharedPreferences preferences;
+    private boolean haspreviouslyStarted;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        preferences = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
+        haspreviouslyStarted = preferences.getBoolean(getString(R.string.preferences_has_previously_started), false);
+        ImageButton new_user = (ImageButton) findViewById(R.id.newUserButtonFirstPage);
```

Comment:
Always use camelCase for Java variables. Here this would be `newUserButton`. Also always choose a name that makes it clear what the variable holds without needing to go to the declaration